test/common/ContractInteractor.test.ts

```diff
@@ -1,27 +1,28 @@
 import chai from 'chai'
 import chaiAsPromised from 'chai-as-promised'
 
-import { RelayHubInstance, TestVersionsInstance } from '../../types/truffle-contracts'
-import { RelayClient } from '../../src/relayclient/RelayClient'
+import { RelayHubInstance } from '../../types/truffle-contracts'
 import { HttpProvider } from 'web3-core'
 import { ProfilingProvider } from '../../src/common/dev/ProfilingProvider'
-import ContractInteractor from '../../src/relayclient/ContractInteractor'
-import { configureGSN } from '../../src/relayclient/GSNConfigurator'
+import ContractInteractor from '../../src/common/ContractInteractor'
 import { PrefixedHexString } from 'ethereumjs-tx'
 import Transaction from 'ethereumjs-tx/dist/transaction'
 import { constants } from '../../src/common/Constants'
 import { createClientLogger } from '../../src/relayclient/ClientWinstonLogger'
 import RelayRequest from '../../src/common/EIP712/RelayRequest'
 import { deployHub } from '../TestUtils'
+import VersionsManager from '../../src/common/VersionsManager'
+import { gsnRuntimeVersion } from '../../src/common/Version'
 
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
```

Comment:
if you don't use expect, don't define it. just do chai.use

Reply:
Sorry, did not commit a  part of a file. This eslint-disable is completely redundant